class/defaults.yml

```diff
@@ -1,3 +1,7 @@
 parameters:
   openshift4_version:
-    namespace: syn-openshift4-version
+    namespace: syn
```

Comment:
I'd propose to get rid of this parameter since it's of no use and can't be changed anyway? (And hard code it to `syn` in the app.yml)